packages/react/examples/nextjs/pages/index.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+// import Head from 'next/head'
+import '@carbon/ibmdotcom-styles/scss/ibm-dotcom-styles.scss';
+import { DotcomShell } from '@carbon/ibmdotcom-react';
+
+/**
+ * Initializes the homepage
+ */
+const Home = () => <DotcomShell navigation="default"></DotcomShell>;
```

Comment:
Should we add any child content for the DotcomShell? otherwise it'll throw an error since I think children is required

Reply:
Also I think I ran into eslint errors earlier after I pulled your branch and built and tested - should we include the build folder in the eslintignore file?

Reply:
Sure, I can add that in.

Reply:
Added with https://github.com/carbon-design-system/ibm-dotcom-library/pull/668/commits/52dd158426410cedde6e9835004b510f990bb9e0